src/components/CardList/CardList.css

```diff
@@ -2,4 +2,40 @@
   display: flex;
   list-style: none;
   flex-wrap: wrap;
-}
\ No newline at end of file
+  padding: 0;
+  margin: 0;
+  justify-content: space-around;
+}
+
+@media screen and (max-width: 1279.5px) {
+  .card-list {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+    grid-row-gap: 90px;
+    grid-column-gap: 78px;
+  }
+  
+  .card-list_type_team {
+    grid-column-gap: calc(100vw * 30/768);
```

Comment:
Could you please explain math expression. What is 30/768?

Reply:
30px is the width of the screen at 768px screen resolution, so the column gap is a steady ratio